exercise/serializer.py

```diff
@@ -15,15 +15,17 @@ class Meta:
         fields = ('id', 'reply_type', 'message')
 
 
-class ExerciseAttachmentSerializer(serializers.HyperlinkedModelSerializer):
+class ExerciseFileSerializer(serializers.HyperlinkedModelSerializer):
+    file_url = serializers.CharField(source='img_url')
```

Comment:
Why not call it either `file_url` in the ExerciseFile model or use` img_url` here in the serialiser that way we don't have to do this referencing since its in the Exercise file model already.

Reply:
Thanks and I agree with you.

For now though we'd need to just match the JSON blob that was designed early days (since the FE expects that).
I wouldn't want to call it `file_url` in the model because someone might think that's the link to the actual file itself (for download or something). However, in reality that's probably going to be the screenshot of the file, hence the name `img_url`.

Been talking to @oluomoniyi - we'll tidy this one up when we have a bit more time over the coming weeks.

Reply:
Cool yeah makes sense. Thanks for clarifying. 👍